src/components/common/TmBalance.vue

```diff
@@ -180,76 +193,97 @@ export default {
         }
       },
       skip() {
+        /* istanbul ignore next */
         return !this.address
       }
     },
-    stakingDenom: {
+    balances: {
       query: gql`
-        query Network($networkId: String!) {
-          network(id: $networkId) {
-            id
-            stakingDenom
+        query balances($networkId: String!, $address: String!) {
+          balances(networkId: $networkId, address: $address) {
+            denom
+            amount
           }
         }
       `,
       variables() {
+        /* istanbul ignore next */
         return {
-          networkId: this.network
+          networkId: this.network,
+          address: this.address
         }
       },
-      update(data) {
+      skip() {
         /* istanbul ignore next */
-        return data.network.stakingDenom
+        return !this.address
       }
     },
-    totalFiatValue: {
+    balancesWithFiat: {
```

Comment:
is it correct, that we load the balances twice now?

Reply:
I don't see any other option. Unless there is some way to use one fragment or another conditionally (either
```
{
amount
denom
}
```
or 
```
{
amount
denom
fiatValue
}
```

Reply:
?! why not just load the latter? it holds all the info right?

Reply:
because I don't have fiatCurrency at the beginning... 

Well, this is easily fixed by selecting a default fiatCurrency. And then I can also make `balancesWithFiat` a computed prop :thinking: 

I will go this way :v: